Error: failed external call: CWE191_Integer_Underflow__int_rand_multiply_68b_badSink
Stack: 
	#000000651 in CWE191_Integer_Underflow__int_rand_multiply_68_bad()
	#100000723 in __user_main()
	#200000439 in __uClibc_main(131791926902784, 1, 131806657298432, 0, 0, 0, 0) at libc/misc/internals/__uClibc_main.c:401
	#300000611 in main(1, 131806657298432)
